src/compat/posix/fs/dvfs/chmod.c

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * @brief
+ *
+ * @date 2.02.2016
+ * @author Alex Kalmuk
+ */
+
+#include <sys/stat.h>
+#include <errno.h>
+#include <fs/dvfs.h>
+
+int chmod(const char *path, mode_t mode) {
```

Comment:
> Also I don't see where information is being updated on the media. 

This is explained in PR's description - currently there is no sync with external storage. Let's suppose for now to use tmpfs or some similar thing.